Makefile

```diff
@@ -55,31 +48,35 @@ SYSCONFDIR:=$(DEFAULT_SYSCONFDIR)
 endif
 SYSCONFDIR:=$(subst PREFIX,$(PREFIX),$(SYSCONFDIR))
 
-export APPLY_PATCHES
+ifndef NEW_BUILD
+NEW_BUILD=$(DEFAULT_NEW_BUILD)
```

Comment:
PULL_ARTIFACTS=$(DEFAULT_PULL_ARTIFACTS)
if enabled, this will pull from nexus/ (in future artifactory), else will locally build?